app/webpacker/components/Panel/SeniorDelegate/Regions/Subregion.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import {
+  Header, Table, Button, Modal, Form, Message,
+} from 'semantic-ui-react';
+import { apiV0Urls, editUserAvatarUrl } from '../../../../lib/requests/routes.js.erb';
+import useLoadedData from '../../../../lib/hooks/useLoadedData';
+import Loading from '../../../Requests/Loading';
+import Errored from '../../../Requests/Errored';
+import WcaSearch from '../../../SearchWidget/WcaSearch';
+import I18n from '../../../../lib/i18n';
+import useSaveAction from '../../../../lib/hooks/useSaveAction';
+
+const delegateStatusOptions = ['trainee_delegate', 'candidate_delegate', 'delegate'];
+const delegateStatusOptionsList = delegateStatusOptions.map((option) => ({
+  text: I18n.t(`enums.user.role_status.delegate_regions.${option}`),
+  value: option,
+}));
+const initialValue = {
+  newDelegate: null,
+  status: delegateStatusOptions[0],
+};
+
+export default function Subregion({ title, groupId }) {
+  const {
+    data: delegates, loading, error: delegatesFetchError, sync,
+  } = useLoadedData(apiV0Urls.userRoles.listOfGroup(
+    groupId,
+    'location,name',
+    {
+      isActive: true,
+      isLead: false,
+    },
+  ));
+  const [openModalType, setOpenModalType] = useState(null);
```

Comment:
Are there going to be other types of modals added to this component in the future?